### PR TITLE
Routing/2s2e/changes to routing

### LINKED ISSUE #46: Changes to Routing
- Resources Page leads to Have Your Voice Heard
- Profile button when signed out leads to sign in instead of saying "sign in to access"

### PR DESCRIPTION
**Pull request by 2s2e**
### Tracking Info

Resolves #46 

### Changes
-Fixed a bunch of bugs to do with header widgets not having the proper parameters
-Profile button on bottom bar, when signed out, takes user to the sign in page
-Back button on have your voice heard goes to the previous screen so that it works both when signed in and signed out
-Resources, when signed in, goes to have your voice heard

### Testing

How did you confirm your changes worked? 

Features were tested both signed in and signed out:
When signed out,
-clicking on the profile button redirects to the sign in page

When signed in,
-clicking on resources brings user to have your voice heard
-back button on have your voice goes back to home page

### Confirmation of Change 

When signed out:
-Click on the profile button to confirm it goes to sign in

When signed in:
-First sign in by creating an account
-After signing in, click on resources, which should redirect to have_your_voice_heard
-Click the back button to confirm the app redirects back to the home page

### CODE REVIEW

File: lib/pages/call_text_now.dart [hunk 1]
```diff
@@ -46,8 +46,11 @@ class _CallTextNowState extends State<CallTextNow> {
       appBar: Header(onBackButtonPressed: () {
         print("back button was pressed from calltext");
       }),
-      bottomNavigationBar: const CustomNavBar(
-        currentPage: 'Profile',
+      bottomNavigationBar: CustomNavBar(
+        currentPage: 'Seek Help',
+        seekHelpRoute: '/callTextNow',
+        profileRoute: '/profile',
+        resourcesRoute: '/haveYourVoiceHeard',
```
Comment (petabite) [inline lib/pages/call_text_now.dart:53]: should this be the coming soon page?
```suggestion
        resourcesRoute: '/comingSoonResources',
```

**Review by petabite — CHANGES_REQUESTED** [review]
nice!

**Review by petabite — APPROVED** [review]
👌🏼

**Review by aaronchan32 — APPROVED** [review]
Looks good to me!

**Pull request merged**